$NetBSD: patch-aa,v 1.4 2010/05/16 09:12:30 obache Exp $

Do not install executables, those are part of ruby-json-pure

--- ../json-1.4.3.gemspec.orig	2010-05-16 09:01:44.000000000 +0000
+++ ../json-1.4.3.gemspec
@@ -10,14 +10,12 @@ bindir: bin
 cert_chain: []
 
 date: 2010-05-04 22:00:00 +00:00
-default_executable: edit_json.rb
+default_executable: []
 dependencies: []
 
 description: This is a JSON implementation as a Ruby extension in C.
 email: flori@ping.de
-executables: 
-- edit_json.rb
-- prettify_json.rb
+executables: []
 extensions: 
 - ext/json/ext/generator/extconf.rb
 - ext/json/ext/parser/extconf.rb
